resources/assets/js/translations/locales/defaultMessages.json

```diff
@@ -1854,6 +1854,16 @@
     ],
     "path": "resources/assets/js/components/JobTasks/JobTasks.json"
   },
+  {
+    "descriptors": [
+      {
+        "defaultMessage": "Your job is loading...",
```

Comment:
This message is currently repeated five times across the various JPB pages, can we centralize it somewhere?

Reply:
Yeah, I'll take care of this.